versions.yaml

```diff
@@ -190,8 +190,8 @@ externals:
 
   attestation-agent:
     description: "Provide attested key unwrapping for image decryption"
-    url: "https://github.com/confidential-containers/attestation-agent"
-    version: "v0.3.0"
+    url: "https://github.com/fitzthum/attestation-agent"
+    version: "snp-attester"
```

Comment:
I'm not sure if you're trying to do this only for testing purposes, then it's okay, but we won't switch the whole project to using the local repo and a specific branch of the attestation-agent.

What I reocmmend is getting that branch merged into the attestation-agent repo, and once that happens we proceed with this PR.